src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -427,8 +427,11 @@ protected override void BeginProcessing()
             }
             else
             {
-                WriteObject(string.Format(CultureInfo.InvariantCulture, "<?xml version=\"1.0\" encoding=\"{0}\"?>", Encoding.UTF8.WebName));
-                WriteObject("<Objects>");
+                if (string.IsNullOrEmpty(Format))
```

Comment:
This should be replaced with a switch based on the enum value, for the same reason as above.